### PR TITLE
MessageNPC packet

### PR DESCRIPTION
**Pull request by 3vcloud**

### CODE REVIEW

File: Include/GWCA/Packets/StoC.h [hunk 1]
```diff
@@ -102,6 +102,14 @@ namespace GW {
             };
             const uint32_t Packet<MessageServer>::STATIC_HEADER = 0x5E;
 
+            // Deliver chat message (sender is an NPC)
+            struct MessageNPC : Packet<MessageNPC> {
+                uint32_t agent_id;
+                uint32_t type;
+                wchar_t sender_name[32];
```
Comment (ldufr) [inline Include/GWCA/Packets/StoC.h]: Are you sure it's `wchar_t[32]` ? I have `wchar_t[8]`.

Comment (3vcloud) [reply Include/GWCA/Packets/StoC.h]: No I'm not sure, but you're probably right as its a hash anyway. Any way to log the capacity of the strings in your logger, not just the size?

Comment (ldufr) [reply Include/GWCA/Packets/StoC.h]: Sure, but the logger did print something lower than 8, right ?

Comment (3vcloud) [reply Include/GWCA/Packets/StoC.h]: Yeh I got a string length of 4 chars consistently when testing in a few factions missions but didn't know if the hash could be longer for other NPCs

**Pull request merged**